cli/src/test_utils.rs

```diff
@@ -62,3 +65,12 @@ pub fn wait_for_next_epoch_plus_n_slots(rpc_client: &RpcClient, n: u64) -> (Epoc
         }
     }
 }
+
+/// Runs local faucet for tests, with non-overlapping ports.
+pub fn run_local_faucet(keypair: Keypair) -> SocketAddr {
```

Comment:
This function should be gated by DCOU as well. This means you'll need to add the DCOU feature to this crate; here is an example from `solana-core`:
https://github.com/anza-xyz/agave/blob/19421efd9118c89e2d45d02bd277025736360e74/core/Cargo.toml#L16-L21